packages/client/bin/cli.ts

```diff
@@ -127,6 +129,16 @@ const args = require('yargs')
       describe: 'Use v4 ("findneighbour" node requests) for peer discovery',
       boolean: true,
     },
+    mine: {
+      describe: 'Enable private custom network mining (beta, PoA only)',
+      boolean: true,
+      default: false,
+    },
+    unlock: {
+      describe:
+        'Comma separated list of accounts to unlock - currently only the first account is used for sealing mined blocks. Beta, you will be promped for a 0x-prefixed private key until keystore functionality is added',
```

Comment:
Do we also want to add the warning to this prompt you wrote somewhere in the PR description, maybe even bold: "(DO NOT USE WITH ACCOUNTS HOLDING SUBSTANTIAL AMOUNTS OF ETH)"?

Reply:
yeah, good suggestion, will do